data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -248,11 +250,47 @@ public static ClientRequestProperties fromString(String json) throws JsonProcess
     }
 
     public String getClientRequestId() {
-        return (String) getOption(OPTION_CLIENT_REQUEST_ID);
+        return _clientRequestId;
     }
 
     public void setClientRequestId(String clientRequestId) {
-        setOption(OPTION_CLIENT_REQUEST_ID, clientRequestId);
+        _clientRequestId = clientRequestId;
+    }
+
+    /**
+     * Gets the application name for tracing purposes.
+     * Overrides the application name set in the connection string.
+     * @return The application name.
+     */
+    public String getApplication() {
```

Comment:
get/setXxxForTracing

Reply:
Currently in c# (and other SDKs that have it) it is just called "Application", do you want to change it in all of them?